x/clp/keeper/migrations.go

```diff
@@ -0,0 +1,26 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type Migrator struct {
+	keeper Keeper
+}
+
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+func (m Migrator) MigrateToVer2(ctx sdk.Context) error {
+	pools := m.keeper.GetPools(ctx)
+	// compute swap prices for each pool
```

Comment:
We should initialize the pmtpEpoch as well . 
Set EpochCounter = 0 and BlockCounter = 0